docs/content/preview/api/ysql/the-sql-language/statements/ddl_create_index.md

```diff
@@ -56,7 +56,9 @@ The following table explains some of the differences between creating an index o
 | Parallelizes index loading? | yes | no |
 
 To enable online schema migration for one `CREATE INDEX`, use `CREATE INDEX CONCURRENTLY`.
+
```

Comment:
Actually it feels a bit hard to understand which is the default behavior. I believe we are concurrent by default? Also I believe ysql_disable_index_backfill=true will take precedence over CREATE INDEX CONCURRENTLY? Would be good to mention that as well.